gems/aws-crt/native/src/credentials.c

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+#include "crt.h"
+
+#include <aws/auth/credentials.h>
+#include <aws/common/string.h>
+
+struct aws_credentials *aws_crt_credentials_new(
+    const char *access_key_id,
+    const char *secret_access_key,
+    const char *session_token,
+    uint64_t expiration_timepoint_seconds) {
+
+    struct aws_allocator *allocator = aws_crt_allocator();
+    return aws_credentials_new(
+        allocator,
+        aws_byte_cursor_from_c_str(access_key_id),
+        aws_byte_cursor_from_c_str(secret_access_key),
+        aws_byte_cursor_from_c_str(session_token),
+        expiration_timepoint_seconds);
+    //    return aws_credentials_new_from_string(
+    //        allocator,
```

Comment:
trivial: delete commented out code

Reply:
Deleted.  I left in place to remind myself to confirm that those strings were being copied.  They are - in aws_credentials_new.  Just to confirm: that makes sense and is correct here right?

Reply:
yes, they're copied in aws_credentials_new()